webapp/channels/src/components/popout_controller/popout_controller.tsx

```diff
@@ -25,7 +26,10 @@ import './popout_controller.scss';
 const PopoutController: React.FC<RouteComponentProps> = (routeProps) => {
     const dispatch = useDispatch();
     const currentUserId = useSelector(getCurrentUserId);
+
     useBrowserPopout();
+    useUserTheme();
```

Comment:
Ideally, this would also take care of adding and removing the `app__body` class like we're doing below, but currently, plugins are unthemed by default, and I didn't feel like risking breaking those right now